easybuild/easyconfigs/k/kallisto/kallisto-0.43.1-foss-2016b.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+
+easyblock = 'CMakeMake'
+
+name = 'kallisto'
+version = '0.43.1'
+
+homepage = 'http://pachterlab.github.io/kallisto/'
+description = """kallisto is a program for quantifying abundances of transcripts from RNA-Seq data, or more generally
+ of target sequences using high-throughput sequencing reads."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://github.com/pachterlab/kallisto/archive/']
+sources = ['v%(version)s.tar.gz']
+
+checksums = ['7baef1b3b67bcf81dc7c604db2ef30f5520b48d532bf28ec26331cb60ce69400']
+
+builddependencies = [
+    ('CMake', '3.7.2')
+]
+
+dependencies = [
+    ('HDF5', '1.8.18', '-serial')
+]
```

Comment:
@boegel I tried it and got a lot of errors durring build due to undefined references to MPI functions, so this might be the reason for the old config using a serial `HDF5`. What would you advise?